trectools/trec_eval.py

```diff
@@ -509,6 +509,9 @@ def get_ndcg(self, depth=1000, per_query=False, trec_eval=True, removeUnjudged=F
         idcg_per_query = perfect_ranking[["query",label]].groupby("query").sum()
         ndcg_per_query = dcg_per_query / idcg_per_query
 
+        #NDCG is 0 if IDCG=0
+        ndcg_per_query.fillna(0.0)
```

Comment:
This should have been:
ndcg_per_query = ndcg_per_query.fillna(0.0)
or 
ndcg_per_query.fillna(0.0, inplace=True)

Could you please fix that?
Thanks

Reply:
@joaopalotti sure, my bad, [fixed it here].(https://github.com/joaopalotti/trectools/pull/25/commits/b6ddb8214ca81fa57de5514cf9ae00cc64ea98f3)